src/harness/harnessLanguageService.ts

```diff
@@ -220,6 +220,10 @@ namespace Harness.LanguageService {
             return !!this.typesRegistry && this.typesRegistry.has(name);
         }
 
+        getGlobalTypingsCacheLocation() {
+            return "/Library/Caches/typescript";

```

Comment:
Anyone have a less kludgy idea for how to test this? I looked into the server fourslash tests, but it would ultimately require the same kind of hard-coded hack, just buried deeper in the infrastructure.